wasm-engine/README.md

```diff
@@ -0,0 +1,84 @@
+# Wasm Engine
+
+## Introduction
+
+This project is a simple WASM cross compilation of the Yggdrasil engine.
+
+Currently, this only supports direct access to the Yggdrasil rule grammar, and is not a full implementation of the Unleash logic.
+
+Please note that this is an experimental project and the API is subject to change rapidly as we iterate on the ideas here.
+
+## Usage
+
+First, install the package:
+
+```sh
+$ yarn add @unleash/yggdrasil-engine
+```
+
+Then, you can use it in your code:
+
+```js
+  const yggdrasil = require("../pkg/wasm_engine.js");
```

Comment:
Is this correct? Or should it be 
```suggestion
  const yggdrasil = require("../pkg/wasm-engine.js");
```
I'm saying because that's the package name but I can't find where we make it snake case

Reply:
Good catch but this is compiler magic that does the case translation. This is just a placeholder so I can do a build pipeline in the next pass, apologies, should have pointed that out